packages/@react-aria/tooltip/src/useTooltipTrigger.ts

```diff
@@ -91,7 +91,9 @@ export function useTooltipTrigger(props: TooltipTriggerProps): TooltipTriggerAri
       'aria-describedby': tooltipId,
       onKeyDown: chain(triggerProps.onKeyDown, onKeyDownTrigger),
       onPress: triggerType === 'click' ? onPress : undefined,
-      ...(triggerType === 'hover' && hoverProps)
+      ...(triggerType.includes('hover') && hoverProps),
+      onFocus: triggerType.includes('focus') ? handleDelayedShow : undefined,
```

Comment:
should all hovers be accompanied by focus triggers? what about all focus triggers, should they be accompanied by a hover?
`onFocus: (triggerType.includes('focus') || triggerType.includes('hover')) ? handleDelayedShow : undefined,`

asking because I think all hoverable things need to be accessible by focus? I honestly don't know, tagging @majornista as well

Reply:
Any tooltip accessible on hover should also be accessible using focus. There really should not be any triggerType 'hover' without 'focus'.

Reply:
Tab around in Slack for a positive example of supporting both hover and focus.

Reply:
I think Rob's suggestion is sufficient, lmk if you disagree